scripts/exgdas_global_marine_analysis_post.py

```diff
@@ -23,7 +23,9 @@
 import shutil
 import logging
```

Comment:
do we need `logging` here still? Or can it be removed like the others?

Reply:
yes, good catch